MonoGame.Framework/Matrix.cs

```diff
@@ -468,7 +637,12 @@ public static void Add(ref Matrix matrix1, ref Matrix matrix2, out Matrix result
 
         }
 
-
+        /// <summary>
+        /// Creates a new <see cref="Matrix"/> which contains the rotation moment around specified axis.
+        /// </summary>
+        /// <param name="axis">The axis of rotation.</param>
+        /// <param name="angle">The angle of rotation.</param>

```

Comment:
Whenever we have an angle parameter we need to specify if it is in radians or degrees.  A very common bug that occurs when the docs are not specific.